src/components/card/card-column/card-column.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import CardColumn from "./card-column.component";
+
+test("renders children within CardColumn as expected", () => {
+  const content = <div>content</div>;
+  render(<CardColumn>{content}</CardColumn>);
+  expect(screen.getByText("content")).toBeInTheDocument();
+});
+
+test("has the expected data attributes when they are passed in", () => {
```

Comment:
question: Do we need to have two renders here? The below would cover it I think:

```
test("has the expected data attributes when they are passed in", () => {
render(
    <CardColumn data-element="foo" data-role="bar">
      <div>content</div>
    </CardColumn>
  );
  
const cardColumnElementWithAttributes = screen.getByTestId("bar"); // data-element is implicitly asserted but we can explicitly if needed
  expect(cardColumnElementWithAttributes).toHaveAttribute(
    "data-component",
    "card-column"
  );
  expect(cardColumnElementWithAttributes).toHaveAttribute(
    "data-element",
    "foo"
);
});